spec/rails_helper.rb

```diff
@@ -77,6 +77,12 @@ def get(path, headers: nil, sign_with: nil, **args)
   config.include Redisable
   config.include SignedRequestHelpers, type: :request
 
+  config.around(:each, use_transactional_tests: false) do |example|
```

Comment:
Part of the maintenance script has a sql statement which uses `CONCURRENTLY` in an index creation. This cannot run inside the transaction that the spec example would normally be running in, hence this around hook.